### PR TITLE
ZXL-188: Apollo client update to 3

### PR DESCRIPTION
**Pull request by yashrajpchavda**

### CODE REVIEW

**Review by itsmeprashant — APPROVED** [review]

File: src/apollo/local-batch-link.ts [hunk 1]
```diff
@@ -1,5 +1,10 @@
-import { ApolloLink, FetchResult, Observable, Operation } from 'apollo-link';
-import { BatchLink } from 'apollo-link-batch';
+import {
+	ApolloLink,
+	FetchResult,
+	Observable,
+	Operation
+} from '@apollo/client/core';
```
Comment (silentsakky) [inline src/apollo/local-batch-link.ts:6]: import from @apollo/client

File: src/apollo/offline-queue-link/index.ts [hunk 2]
```diff
@@ -1,4 +1,9 @@
-import { ApolloLink, NextLink, Observable, Operation } from 'apollo-link';
+import {
+	ApolloLink,
+	NextLink,
+	Observable,
+	Operation
+} from '@apollo/client/core';
```
Comment (silentsakky) [inline src/apollo/offline-queue-link/index.ts:6]: import from @apollo/client

File: src/apollo/offline-queue-link/types.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { FetchResult, NextLink, Operation } from 'apollo-link';
+import { FetchResult, NextLink, Operation } from '@apollo/client/core';
```
Comment (silentsakky) [inline src/apollo/offline-queue-link/types.ts:1]: import from @apollo/client

File: src/apollo/offline-queue-link/util.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { Operation } from 'apollo-link';
+import { Operation } from '@apollo/client/core';
```
Comment (silentsakky) [inline src/apollo/offline-queue-link/util.ts:1]: import from @apollo/client

Comment (silentsakky) [reply src/apollo/offline-queue-link/util.ts]: Fixed

File: src/apollo/offline-queue-link/types.ts (hunk 3 above)
Comment (silentsakky) [reply src/apollo/offline-queue-link/types.ts]: Fixed

File: src/apollo/offline-queue-link/index.ts (hunk 2 above)
Comment (silentsakky) [reply src/apollo/offline-queue-link/index.ts]: Fixed

File: src/apollo/local-batch-link.ts (hunk 1 above)
Comment (silentsakky) [reply src/apollo/local-batch-link.ts]: Fixed

File: rollup.config.js [hunk 5]
```diff
@@ -15,8 +15,8 @@ let external = FORMAT==='es' ?
 	Object.keys(pkg.dependencies)
 		.concat(
 			['castArray', 'get','isError', 'isObject', 'mapValues', 'reduce', 'omitBy', 'uniqBy', 'concat', 'uniqBy', 'differenceBy', 'forEach'].map(v => 'lodash/'+v),
-			['graphql/language/printer', 'graphql/type']) :
-	['graphql-tools'];
+			['graphql/language/printer', 'graphql/type', '@graphql-tools/schema']) :
```
Comment (silentsakky) [inline rollup.config.js:18]: `'graphql/language/printer', 'graphql/type' ==> 'graphql'`

**Review by itsmeprashant — APPROVED** [review]

**Review by lavegupta — APPROVED** [review]

**Review by auminpatel — APPROVED** [review]

**Pull request merged**